R/panel.R

```diff
@@ -0,0 +1,10 @@
+#' Creates a panel for all info boxes so they do not overlap
+#'
+#' @param position character with position of the parameter. Default "top right".
+#' @param ... calls with info elements
+#'
+#' @return div which wraps your all info boxes to display it in the {position} corner of your shiny app.
+#' @export
+infoPanel <- function(position = "top left", ...) {
```

Comment:
Let's make it default top right and change the order: `function(..., position = "top right")`